docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "Graph panel"
+keywords = ["grafana", "graph panel", "documentation", "guide", "graph"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/graph/", "/docs/grafana/latest/features/panels/graph/"]
+[menu.docs]
+parent = "visualizations"
+weight = 100
+draft = "true"
++++
+
+# Graph panel
+
+This visualization is the most-used in the Grafana ecosystem. It can render as a line, a path of dots, or a series of bars. This type of graph is versatile enough to display almost any time-series data.
+
+## Data and field options
+
+Graph visualizations allow you to apply:
+- Data transformations
+- Alerts - This is the only type of visualization that allows you to set alerts. For more information, see [LINK TO UPDATED ALERTING TOPIC]
+- [Thresholds]({{< relref "../thresholds.md" >}})
+
+## Display options
+
+Use these settings to refine your visualization.
+
+- **Bars -** Display values as a bar chart.
+- **Lines -** Display values as a line graph.
+- **Line width -** The width of the line for a series. (default 1).
+- **Staircase -** Draws adjacent points as staircase.
+- **Area fill -** Amount of color fill for a series. (default 1, 0 is none)
+- **Fill gradient -** XXX
+- **Points -** Display points for values.
+- **Point radius -** Controls how large the points are.
+
+### Hover tooltip
+XXX - this entire section
+- **Mode**
+  - **All series -**
+  - **Single -**
+- **Sort order**
+  - **None -**
+  - **Increasing -**
+  - **Decreasing -**
+
+### Stacking and null value
+
+- **Stack -** Each series is stacked on top of another.
+- **Percent -** Available when **Stack** is selected. Each series is drawn as a percentage of the total of all series.
+- **Null value -** How null values are displayed.
+  - **connected -** XXX
+  - **null -** XXX
+  - **null as zero -** XXX
+
+## Series overrides
+
+Series overrides allows a series in a graph panel to be rendered differently from the others. You can customize display options on a per-series bases or by using regex rules. For example, one series can have a thicker line width to make it stand out or be moved to the right Y-axis.
+
+You can add multiple series overrides.
+
+**Add a series override**
+
+1. Click **Add series override**.
+1. In **Alias or regex** Type or select a series. Click in the field to see a list of available series.
```

Comment:
Maybe an example of a regex would be fitting. `/Network.*/` would match two series named `Network out` and `Network in`.